src/dispatch.c

```diff
@@ -14,7 +14,8 @@ Rboolean should_ignore(SEXP value, SEXP ignore) {
   return FALSE;
 }
 
-SEXP method_internal(SEXP table, SEXP signature, R_xlen_t signature_itr, SEXP ignore) {
+// Recursively walk through method table to perform iterated dispatch
+SEXP method_rec(SEXP table, SEXP signature, R_xlen_t signature_itr, SEXP ignore) {
```

Comment:
I feel like I will always forget that `rec` is short for recurse, could it just be `method_recurse()`? Same comment about the R function that has a similar name